.env

```diff
@@ -0,0 +1,31 @@
+TYPE="source" # OR "channel"
```

Comment:
This replaces the former `.up_env`, is used both in `up.sh` and `down.sh` and overwrites the new sandbox `.env` during `up.sh`